common/package.json

```diff
@@ -12,31 +12,24 @@
     "@elastic/apm-rum": "^5.12.0",
     "@next/bundle-analyzer": "^13.2.3",
     "@popperjs/core": "^2.11.7",
-    "@prismicio/client": "^7.3.1",
+    "@prismicio/client": "7.6.0",
```

Comment:
[Versions after that were creating issues](https://github.com/prismicio/prismic-client/issues/345), so I pinned it to the recommended version.

Reply:
Hi @rcantin-w, this should be fixed with `@prismicio/client` latest release, you should be able to keep using latest (`^7.8.0`)

Reply:
Thank you @lihbr! I'll try it now 🙏

Reply:
Amended in [this commit](https://github.com/wellcomecollection/wellcomecollection.org/pull/11068/commits/404f2647eb094216ea8d6e98bb6b5e58adbc3e52) and modified PR description accordingly.